odkx-src/build-app.rst

```diff
@@ -66,19 +66,21 @@ When creating a new form, the appropriate directory structure must be created. O
 Creating the Directory Structure
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-New forms must be placed under the :file:`app/config/tables/` directory as described in the :ref:`app-designer-dirs-app-config-tables` section. Given a form with the name *formId*, it will have a *tableId* of the same name unless you explicitly specify otherwise. The directory structure that should be created is :file:`app/config/tables/tableId/forms/formId` (where, under many circumstances, the value for *tableId* will be the same as the value for *formId*). To get started, for Windows open a :program:`cmd` window within your :file:`Application Designer` folder (click the :program:`cmd` shortcut you created earlier), and for Mac/Unix open a :program:`terminal` window within your :file:`Application Designer` folder. Type:
+New forms must be placed under the :file:`app/config/tables/` directory as described in the :ref:`app-designer-dirs-app-config-tables` section. Given a form with the name *formId*, it will have a *tableId* of the same name unless you explicitly specify otherwise. The directory structure that should be created is :file:`app/config/tables/tableId/forms/formId` (where, under many circumstances, the value for *tableId* will be the same as the value for *formId*). 
 
-.. code-block:: console
-
-  $ grunt addtable:tableId
+To get started: 
 
-Where tableId is the name of your new form and table. For example, to create a census form, type:
-
-.. code-block:: console
+  1. Navigate to :file:`app/config/tables/` and create a folder with the tableId, where tableId is the name of your new form and table. For example, to create a census form; the folder would be named census.
 
-  $ grunt addtable:census
+  2. In the census folder, create the following new folders:
+   
+    - :th:`forms`
+    - :th:`html`
+    - :th:`js`
+    - :th:`instances`
+    - :th:`collect-forms`
```

Comment:
I am not sure all these folders are needed... forms definitely. Are the others needed for tables? I mostly use survey so not sure.